src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -118,17 +118,34 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="InitialJsonParseErrorText" xml:space="preserve">
-    <value>Settings could not be loaded from file - temporarily using the default settings. Check for syntax errors, including trailing commas.</value>
+    <value>Settings could not be loaded from file - temporarily using the default settings. Check for syntax errors, including trailing commas.
+</value>
+  </data>
+  <data name="MissingDefaultProfileText" xml:space="preserve">
+    <value>Could not find your default profile in your list of profiles - temporarily using the first profile. Check to make sure the defaultProfile matches the guid of one of your profiles.
+</value>
+  </data>
+  <data name="DuplicateProfileText" xml:space="preserve">
+    <value>Found multiple profiles with the same guid in your profiles - temporarily ignoring duplicates. Make sure each profile's guid is unique.

```

Comment:
should we capitalize `GUID`?

Reply:
Usually we do, but at the same time, the key is `guid` not `GUID`, so IDK. Maybe we should have a wordsmith's opinion @bitcrazed @cinnamon-msft

Reply:
Most of the projects I've seen follow much of the guidance in [Google's JSON Style Guide](https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format) which states that property names should be camel cased:

    {
      "thisPropertyIsAnIdentifier": "identifier value"
    }

Reply:
sooooooooo you're saying we should call it `gUID`?

Reply:
@bitcrazed This is not a question about our json schema. This is a question about a user-facing error message

Reply:
Ohhhhh - apologies! 

In human-text, acronyms formed from the initial letter of each word should be all-caps - GUID.

Reply:
I agree with GUID